addons/binding/org.openhab.binding.onebusaway/src/main/java/org/openhab/binding/onebusaway/OneBusAwayBindingConstants.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
```

Comment:
It seems that this file has a missing  or invalid copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.